src/components/04-tables/tables.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * JSX Tables Component Attributes
+ * @function Tables
+ * @param {Object} config - The configuration object for table attributes
+ * @property {string} config.name  - The name of the table.
+ * @property {string} config.class - The classes inherited from the USWDS CSS files. The default class is `usa-table` and additions can be appended.
+ * @property {Array} config.children - When this template is used to make a react component @babel/core will change "config" to "props" so that children components can be accessed.
+ * @property {string} config.id    - The ID of the table
+ * @example
+ *   const config = {
+ *     name: "Table",
+ *     class: "usa-table--borderless",
+ *     id: "813"
+ *   };
+ *   Tables(config)
+ * @returns {HTMLTableElement} The HTML Table
+ */
+exports.Tables = function (config) {
+  return (
+    <table id={`${config.tableID}`} class={`usa-table ${config.class}`}>
```

Comment:
non-blocker, but is there a way to infer or generate an id so that it doesn't need to be passed in? are there problems that arise if it is inferred or generated?

Reply:
There's probably a random number generator JavaScript has that I could use to create one.